tests/modules/programs/kitty/theme-to-themeFile.nix

```diff
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, options, ... }: {
+  config.programs.kitty = {
+    enable = true;
+    theme = "Space Gray Eighties";
+
+    test.stubs.kitty = { };
+
+    test.asserts.warnings.enable = true;
+    test.asserts.warnings.expected = [''
```

Comment:
This test is failing locally for me with this message

> Expected home-files/asserts/warnings.actual to be same as /nix/store/arjnippklyvws49lxwgi1y7hb0yjayh5-warnings.expected but were different:

I checked the expected warnings file and it's empty for some reason.

```
❯ bat /nix/store/arjnippklyvws49lxwgi1y7hb0yjayh5-warnings.expected
───────┬──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: /nix/store/arjnippklyvws49lxwgi1y7hb0yjayh5-warnings.expected   <EMPTY>
───────┴──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

```

Need to look into this later

Reply:
Ah this was nested under config.programs.kitty :facepalm: